engine/src/main/java/io/zeebe/engine/processing/streamprocessor/StreamProcessor.java

```diff
@@ -122,6 +122,10 @@ protected void onActorStarted() {
       final ReProcessingStateMachine reProcessingStateMachine =
           new ReProcessingStateMachine(processingContext);
 
+      // disable writing to the log stream but for reprocessing checks
+      processingContext.disableLogStreamWriter();
+      processingContext.enableReprocessingStreamWriter();
+
```

Comment:
line 127 kind of negates whatever 126 did. Did you want this explicitly to make it more clear what happens? I think it's fine to use only line 127.

Reply:
Yes, exactly. I would just keep it for now. We can look at it again when we remove the reprocessing writer after the migration is done.